README.md

```
@@ -0,0 +1,91 @@
+
+1. **PR 생성**: GitHub에서 `[내 브랜치]` ➡️ `develop` 으로 PR을 생성합니다.
+2. **공유 및 리뷰**: PR 링크를 카톡방에 공유하고 팀원의 리뷰(또는 확인)를 받습니다.
+3. **머지 (중요)**: 리뷰가 완료되면 **[Squash and Merge]** 버튼을 눌러 병합합니다.
+
+
+# Commit message format
+
+> 커밋 메세지의 기본 포멧은 아래의 명세를 따릅니다.
+> 
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)